app/services/importers/create_cohort.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Importers
+  class CreateCohort < BaseService
+    def call
+      check_headers!
+
+      logger.info "CreateCohort: Started!"
+
+      ActiveRecord::Base.transaction do
+        rows.each do |row|
+          create_cohort(row)
+        end
+      end
+      create_next_cohort
+
+      logger.info "CreateCohort: Finished!"
+    end
+
+  private
+
+    attr_reader :path_to_csv, :logger
+
+    def initialize(path_to_csv:, logger: Rails.logger)
+      @path_to_csv = path_to_csv
+      @logger = logger
+    end
+
+    def create_cohort(row)
+      start_year = row["start-year"].to_i
+      logger.info "CreateCohort: Creating cohort for starting year #{start_year}"
+
+      Cohort.find_or_create_by!(start_year:) do |c|
+        c.registration_start_date = safe_parse(row["registration-start-date"])
+        c.academic_year_start_date = safe_parse(row["academic-year-start-date"])
+        c.npq_registration_start_date = safe_parse(row["npq-registration-start-date"])
+      end
+
+      logger.info "CreateCohort: Cohort for starting year #{start_year} successfully created"
+    end
+
+    def create_next_cohort
```

Comment:
I wonder if this could be dangerous in production, because we are only expecting to create cohorts according to the CSV only, where this might create the next cohort accidentally in production. Preferably we can avoid mixing dynamic seed data logic with the service to create Cohorts according to a given CSV. 

Maybe the best middle ground is to leave the seeds where we use dynamic calculations as is, as in the logic for dynamic cohorts remains the same and the service class remains as it was before, and only reads from the CSV?
Or we return early in this method if the environment is prod?